lemur/notifications/messaging.py

```diff
@@ -87,29 +89,30 @@ def get_eligible_certificates(exclude=None):
     return certificates
 
 
-def send_notification(event_type, data, targets, notification):
+def send_plugin_notification(event_type, data, recipients, notification):
     """
     Executes the plugin and handles failure.
 
     :param event_type:
     :param data:
-    :param targets:
+    :param recipients:
     :param notification:
     :return:
     """
     function = f"{__name__}.{sys._getframe().f_code.co_name}"
     log_data = {
         "function": function,
-        "message": f"Sending expiration notification for to targets {targets}",
+        "message": f"Sending expiration notification for to recipients {recipients}",
         "notification_type": "expiration",
-        "certificate_targets": targets,
+        "certificate_targets": recipients,
     }
     status = FAILURE_METRIC_STATUS
     try:
-        notification.plugin.send(event_type, data, targets, notification.options)
+        current_app.logger.debug(log_data)
```

Comment:
should this be info?

Reply:
Same concern about volume (but even more than SNS) - I'm imagining there could be dozens/hundreds/maybe thousands of email notifications sent at once, which could be more logging than is actually useful. That said, if you don't think it's a valid concern, I can change it.

Reply:
I see your point. This might be way higher volume than the SNS part.